app/views/posts/index.html.erb~

```diff
@@ -0,0 +1,27 @@
+<p style="color: green"><%= notice %></p>
```

Comment:
Same comment as not showing up as a valid file due to ~ end of the file extension.